src/applications/personalization/profile-2/components/ProfileHeader.jsx

```diff
@@ -100,31 +104,39 @@ const ProfileHeader = ({
         >
           {showBadgeImage && (
             <img
-              className="profile-service-badge"
               src={SERVICE_BADGE_IMAGE_PATHS.get(latestBranchOfService)}
-              alt="service badge"
+              alt={`${latestBranchOfService} seal`}
+              className="profile-service-badge"
+              aria-hidden="true"
```

Comment:
I know this intentional but for my own learning, why is this `aria-hidden`? Just curious how that plays in with the `alt`

Reply:
cuz Jen told me to? I think because it's entirely decorative the screenreaders can just ignore it. I'm not sure how the `alt` fits into this aside from the fact that I `alt` attributes are used for other purposes beyond screen readers?